bin/clock.rb

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env ruby
 require 'clockwork'
 
+UPDATE_TARGETS_RATE_SECONDS = ENV.fetch('UPDATE_TARGETS_RATE_SECONDS', '120').to_i
```

Comment:
This default is smaller than the one in the manifest files (by 60 seconds). Is that intentional? What's the reasoning behind 120 or 180 seconds?

Reply:
The choice behind 120 and 180 is arbitrary, 120 is a nice safe default. I picked 180 because it is a substantial reduction that still feels acceptable to a user pushing a new app.

Happy to change if people have opinions :D